documentation/community/previous_and_upcoming_events.md

```diff
@@ -1,9 +1,11 @@
 # Previous and Upcoming Events
 
 ## Upcoming Events
+
```

Comment:
Are these markdown changes supposed to be here?

Reply:
Nope, hoping that if I merge latest with master these will go away

Reply:
@chrismaree these files are getting linted for some reason. I was able to override these by `--no-verify`-ing them. Any idea what's going on?

Reply:
I have not merged the markdown linting. I still want to show Hart and Regina how to do this locally before merging the PR. this wont happen after that.